dev/com.ibm.ws.microprofile.faulttolerance.2.0/src/com/ibm/ws/microprofile/faulttolerance20/state/impl/AsyncBulkheadStateNullImpl.java

```diff
@@ -18,22 +18,23 @@
 public class AsyncBulkheadStateNullImpl implements AsyncBulkheadState {
```

Comment:
Changes to this class just reflect the changes to the `AsyncBulkheadState` interface